pkg/event/target/elasticsearch.go

```diff
@@ -98,25 +98,19 @@ func (target *ElasticsearchTarget) HasQueueStore() bool {
 
 // IsActive - Return true if target is up and active
 func (target *ElasticsearchTarget) IsActive() (bool, error) {
-	if dErr := target.args.URL.DialHTTP(nil); dErr != nil {
-		if xnet.IsNetworkOrHostDown(dErr) {
-			return false, errNotConnected
-		}
-		return false, dErr
-	}
-	return true, nil
+	return target.client != nil, nil
```

Comment:
We need the connection information for functions like `getARNList` and `fetchLambdaInfo` etc. I think this will return `true, nil` in all the cases. 

IMO, We need dials/pings to provide instant info on target's connections

Reply:
Hmm we can preserve them, but we should never use them internally. Because making such as hoc calls for each event is expensive operation.

Reply:
Yeah, We can have the ping in IsActive() function and lets not use it in internal functions like Save(),Send() as this PR suggests